modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -20,69 +20,57 @@ Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone
-** Insights Operator. Enables a new data collection feature called 'Insights Runtime Extractor' which, when enabled, allows Red{nbsp}Hat to gather more runtime workload data about your {product-title} containers. (`InsightsRuntimeExtractor`)
-** Dynamic Resource Allocation API. Enables a new API for requesting and sharing resources between pods and containers. This is an internal feature that most users do not need to interact with. (`DynamicResourceAllocation`)
-** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
-** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
-** Image mode behavior of image streams. Enables a new API for controlling the import mode behavior of image streams. (`imageStreamImportMode`)
-** `OVNObservability` resource allows you to verify expected network behavior. Supports the following network APIs: `NetworkPolicy`, `AdminNetworkPolicy`, `BaselineNetworkPolicy`, `UserDefinesdNetwork` isolation, multicast ACLs, and egress firewalls. When enabled, you can view network events in the terminal.
 ** `gcpLabelsTags`
-** `vSphereStaticIPs`
 ** `routeExternalCertificate`
 ** `automatedEtcdBackup`
 ** `gcpClusterHostedDNS`
-** `vSphereControlPlaneMachineset`
 ** `dnsNameResolver`
 ** `machineConfigNodes`
-** `metricsServer`
-** `installAlternateInfrastructureAWS`
 ** `mixedCPUsAllocation`
 ** `managedBootImages`
 ** `onClusterBuild`
 ** `signatureStores`
 ** `SigstoreImageVerification`
 ** `DisableKubeletCloudCredentialProviders`
 ** `BareMetalLoadBalancer`
-** `ClusterAPIInstallAWS`
-** `ClusterAPIInstallAzure`
-** `ClusterAPIInstallNutanix`
-** `ClusterAPIInstallOpenStack`
-** `ClusterAPIInstallVSphere`
 ** `HardwareSpeed`
 ** `KMSv1`
 ** `NetworkDiagnosticsConfig`
 ** `VSphereDriverConfiguration`
 ** `ExternalOIDC`
 ** `ChunkSizeMiB`
-** `ClusterAPIInstallGCP`
-** `ClusterAPIInstallPowerVS`
 ** `EtcdBackendQuota`
 ** `InsightsConfig`
 ** `InsightsOnDemandDataGather`
 ** `MetricsCollectionProfiles`
 ** `NewOLM`
+** `NewOLMCatalogdAPIV1Metas`
+** `NewOLMOwnSingleNamespace`
+** `NewOLMPreflightPermissionChecks`
 ** `AWSClusterHostedDNS`
 ** `AdditionalRoutingCapabilities`
 ** `AutomatedEtcdBackup`
 ** `BootcNodeManagement`
-** `CSIDriverSharedResource`
+** `ClusterAPIInstallIBMCloud`
 ** `ClusterMonitoringConfig`
 ** `ConsolePluginContentSecurityPolicy`
 ** `DNSNameResolver`
+** `DyanmicServiceEndpointIBMCloud`
 ** `DynamicResourceAllocation`
 ** `EtcdBackendQuota`
 ** `Example`
+** `ExternalOIDCWithUIDAndExtraClaimMappings`
```

Comment:
Can I ask how you got this list? I am expecting `ExternalOIDC` in this list for 4.19, but I'm not sure about this value.

Reply:
So @cbippley can you remove this one and add `ExternalOIDC` in its place?

Reply:
Oh sorry, I see it down further, I didn't realize this list wasn't alphabetical